openmeter/billing/entity/errors.go

```diff
@@ -28,6 +28,8 @@ var (
 	ErrInvoiceActionNotAvailable = NewValidationError("invoice_action_not_available", "invoice action not available")
 
 	ErrInvoiceLineFeatureHasNoMeters             = NewValidationError("invoice_line_feature_has_no_meters", "usage based invoice line: feature has no meters")
+	ErrInvoiceLineGraduatedSplitNotSupported     = NewValidationError("invoice_line_graduated_split_not_supported", "graduated tiered pricing is not supported for split periods")
```

Comment:
should be registered in httpdriver

Reply:
All of these are wrapped into ValidationErrors or whatever error it should be reported as, and that is the thing that the HTTP driver handles.

I can only add a fallback, but regardless the previous logic should ensure proper error codes.